### DISCUSSION TITLE
What is the point of unamed aspects ?

### DISCUSSION THREAD

**Discussion by lesquoyb** [gama-platform/gama#88] [Q&A]
In species you can write aspects without giving them any name, like so:
```
aspect {
draw circle(1);
}
```
But those aspects are not considered as default aspects when referencing the species in a display, and can't also be called using the `aspect` facet because it is not found.
![image](https://user-images.githubusercontent.com/6374469/156104249-ffb3a37c-803f-4126-b000-9d20b741fcd7.png)

So from what I understand those aspects could never be used. Is there any point in keeping it like it is ? Couldn't they be considered as default ?

**Comment by agrignard** [comment]
This aspect can be used if you write only ```species cursor;``` 

Note that if you define an aspect called ```default```you can also use this syntax ```species cursor;```

**Reply by lesquoyb** [reply]
So maybe that's a bug, because with this model:

```
model TestAspect

species cursor skills:[moving] {
	
	aspect {
		draw circle(3) color:#blue;
	}
	
	reflex move {
		do wander;
	}
	
}



experiment UserLocation type: gui {
	
	init {
		create cursor number:1;
	}
	
	output {
		display main {
			species cursor;
		}
	}
	
}
```

I do not see anything, while giving it the `default` name it works as expected.
